MapboxNavigation/PollyVoiceController.swift

```diff
@@ -150,13 +152,14 @@ public class PollyVoiceController: RouteVoiceController {
                         audioPlayer.volume = strongSelf.volume
                         audioPlayer.play()
```

Comment:
the return value of `.play()` is discardable but I think we should observe it.

Reply:
#661 takes care of responding to the return value.